qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -629,7 +636,7 @@
             master_images_clone = image1
             remove_image_image1 = yes
             cmd_run_debugview = 'start WIN_UTILS:\Debugviewconsole.exe -d C:\debug.dbglog'
-            cmd_check_string_VSS = 'type C:\debug.dbglog | findstr /i "QEMU Guest Agent VSS Provider\[[0-9]*\]"'
+            cmd_check_string_VSS = 'type C:\debug.dbglog | findstr /i "QEMU Guest Agent VSS Provider\info[[0-9]*\]"'
```

Comment:
Could you explain more about why we need to modify this cmd value?Seems it it not related to current test case.

Reply:
TBH， I forget the reason, because it's far away when I first started writing. maybe it's type by mistake, or Intentionally do this. anyway, I'll run the test with/without this change and back here to update.

Reply:
alright typo problem. upgraded. thanks.